pkgs/applications/audio/caps/0001-Avoid-ambiguity-in-div-invocation.patch

```diff
@@ -0,0 +1,23 @@
+From: Felipe Sateler <fsateler@debian.org>
```

Comment:
Is it possible to retrieve this from an external source instead of checking it into the repo?

Reply:
done